dev/podman/setup-local-env-podman.md

```diff
@@ -1,54 +1,96 @@
-## The following are Podman specific steps used to set up on a MacBook (Intel or Apple Silicon)
+# Configuring Podman for Tilt
 
-### Verify installed tools (install if needed)
+## Verify installed tools (install if needed)
+
+Ensure you have installed [Podman](https://podman.io/), [Kind](https://github.com/kubernetes-sigs/kind/), and [Tilt](https://tilt.dev/).
 
 ```sh
 $ podman --version
 podman version 5.0.1
 $ kind version
 kind v0.23.0 go1.22.3 darwin/arm64
-
-(optional)
 $ tilt version
 v0.33.12, built 2024-03-28
 ```
 
-### Start Kind with a local registry
+## Start Kind with a local registry
+
 Use this [helper script](./kind-with-registry-podman.sh) to create a local single-node Kind cluster with an attached local image registry.
 
-#### Disable secure access on the local kind registry:
 
-`podman inspect kind-registry --format '{{.NetworkSettings.Ports}}'`
+## Disable secure access on the local kind registry:
 
-With the port you find for 127.0.0.1 edit the Podman machine's config file:
+Verify the port used by the image registry:
 
-`podman machine ssh`
+```sh
+podman inspect kind-registry --format '{{.NetworkSettings.Ports}}'
+```
 
-`sudo vi /etc/containers/registries.conf.d/100-kind.conf`
+If your host is a Mac, first run:
```

Comment:
Since you're broadening the doc scope beyond just macs and into linux processes, it would be nice to have a consistent approach to "{{ if $THISPLATFORM }} dostuff {{ endif }}".  
I'll take a longer look later, but it gets a bit confusing later in this doc where it suggests that a mac could run systemctl. 😄

Reply:
hehe yeah, good catch. i mean if there's enough differences we could just have distinct docs/tutorials for diff OSes